pkg/env/prod.go

```diff
@@ -132,14 +132,12 @@ func newProd(ctx context.Context, log *logrus.Entry) (*prod, error) {
 		return nil, err
 	}
 
-	fpPrivateKey, fpCertificates, err := p.serviceKeyvault.GetCertificateSecret(ctx, RPFirstPartySecretName)
+	p.fpRefreshingManager = newCertificateRefresher(ctx, log, 1*time.Hour, p.serviceKeyvault, RPFirstPartySecretName, stop)
+	err = p.fpRefreshingManager.Start(ctx)
```

Comment:
Again, nit: just more consistent naming.
```suggestion
	p.fpCertificateRefresher = newCertificateRefresher(ctx, log, 1*time.Hour, p.serviceKeyvault, RPFirstPartySecretName, stop)
	err = p.fpCertificateRefresher.Start(ctx)
```